internal/commands/result.go

```diff
@@ -614,7 +625,7 @@ func summaryReport(
 		summary.Policies = filterViolatedRules(*policies)
 	}
 
-	enhanceWithScanSummary(summary, results)
+	enhanceWithScanSummary(summary, results, featureFlagsWrapper)
 
```

Comment:
do we count api security critical severity? i saw that we found it in line 659, right?